earthly-entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+set -e
+
+# Apply global configuration
+if [ -n "$GLOBAL_CONFIG" ]; then
+  earthly config global "$GLOBAL_CONFIG"
+fi
+
+# Apply git configuration
+if [ -n "$GIT_CONFIG" ]; then
+  earthly config git "$GIT_CONFIG"
+fi
+
+# We prefer you use EARTHLY_TOKEN directly, but this is here just in case
+if [ -n "$EARTHLY_EMAIL" ] && [ -n "$EARTHLY_PASSWORD" ]; then
+  earthly account login --email "$EARTHLY_EMAIL" --password "$EARTHLY_PASSWORD"
+fi
```

Comment:
Maybe it's better if we don't support the less secure option at all and just require people to create tokens.

Reply:
Ill drop this then.

Reply:
Don't see why not - just needs to be documented.